app/views/layouts/_head.html.erb

```diff
@@ -36,6 +36,17 @@
   <%= include_account_css %>
   <script>
     window.bz_custom_css_url = <%= custom_css_url.to_json.html_safe %>;
+    try {
+      if(window.location.pathname != '/' && window.location.pathname.indexOf('/login') != 0) {
```

Comment:
This seems brittle.  Are there no built-in variable names for root and login routes?